components/LoginScreen.jsx

```diff
@@ -0,0 +1,156 @@
+import { StatusBar } from "expo-status-bar";
+import { useEffect, useState } from "react";
+import { FontAwesome5 } from "@expo/vector-icons";
+import {
+  KeyboardAvoidingView,
+  StyleSheet,
+  Text,
+  TextInput,
+  TouchableOpacity,
+  View,
+  Image,
+} from "react-native";
+
+import {
+  getAuth,
+  createUserWithEmailAndPassword,
+  signInWithEmailAndPassword,
+} from "firebase/auth";
+import { auth } from "../firebaseConfig";
+import { useNavigation } from "@react-navigation/native";
+
+export default LoginScreen = ({ navigation }) => {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+
+  const navigation1 = useNavigation();
+
+  useEffect(() => {
+    const unsubscribe = auth.onAuthStateChanged((user) => {
+      if (user) {
+        navigation1.navigate("Home");
+      }
+    });
+
+    return unsubscribe;
+  }, []);
+
+  const handleSignUp = () => {
+    createUserWithEmailAndPassword(auth, email, password)
+      .then((userCredentials) => {
+        const user = userCredentials.user;
+        console.log("Registered with", user.email);
+        alert("Account Registered");
+      })
+      .catch((error) => {
+        alert(error.message);
+      });
+  };
+
+  const handleLogin = () => {
+    signInWithEmailAndPassword(auth, email, password)
+      .then((userCredentials) => {
+        const user = userCredentials.user;
+        console.log("Logged in with", user.email);
+        alert("Logged in");
+      })
+      .catch((error) => {
+        alert(error.message);
+      });
+  };
+
+  return (
+    <KeyboardAvoidingView style={styles.container} behavior="padding">
+      <View style={styles.inputContainer}>
+        <Text style={styles.title}>WELCOME TO FEATHER QUEST!</Text>
+        <View style={styles.iconContainer}>
+          <Image source={require("../assets/feather.png")} />
+        </View>
+        <TextInput
+          placeholder="Email"
```

Comment:
We could add an autoCapitalize="none" on these input fields to prevent the user's email being capitalised on entry ie from cam@email.com gets captailised currently to Cam@email.com

Reply:
got it thanks